Makefile

```
@@ -133,7 +133,7 @@ enzyme:
 	$(MAKE) -C mlir enzyme
 
 dialects:
-	$(MAKE) -C mlir dialects
+	$(MAKE) -C mlir dialects dialect-docs
```

We probably don't need to build this every time now since the docs are checked in statically
```suggestion
	$(MAKE) -C mlir dialects
	
.PHONY: dialect-docs
dialect-docs:
	$(MAKE) -C mlir dialect-docs
```